cli/src/semgrep/commands/scan.py

```diff
@@ -348,6 +348,15 @@ def convert(
             skipped. If set to 0 will not have limit. Defaults to 3.
         """,
     ),
+    optgroup.option(
+        "--pro-timeout",
+        type=int,
+        help=f"""
+            Maximum time to spend on preprocessing for pro features. If set to 0
```

Comment:
As I said in the other PR I think this should be "inter-file analysis". Apex is a Pro feature and doesn't need this timeout.